etc/kayobe/environments/ci-aio/stackhpc-ci.yml

```diff
@@ -54,8 +54,8 @@ stackhpc_repo_centos_stream_9_docker_version: "{{ stackhpc_pulp_repo_centos_stre
 stackhpc_repo_rhel_9_treasuredata_4_version: "{{ stackhpc_pulp_repo_rhel_9_treasuredata_4_version }}"
 stackhpc_repo_rhel_9_mariadb_10_6_version: "{{ stackhpc_pulp_repo_rhel_9_mariadb_10_6_version }}"
 stackhpc_repo_rhel_9_influxdb_version: "{{ stackhpc_pulp_repo_rhel_9_influxdb_version }}"
-stackhpc_repo_rocky_9_crb_version: "{{ stackhpc_pulp_repo_rocky_9_1_crb_version }}"
-stackhpc_repo_rocky_9_highavailability_version: "{{ stackhpc_pulp_repo_rocky_9_1_highavailability_version }}"
+stackhpc_repo_rocky_9_crb_version: "{{ stackhpc_pulp_repo_rocky_9_crb_version }}"
```

Comment:
I think you need to replace all `9` by `9_2` in the references to variables.

Reply:
Also this guy: https://github.com/stackhpc/stackhpc-kayobe-config/blob/9e6ffaa949618ff129aceca92498f80cb1879b7e/etc/kayobe/pulp.yml#L216

Reply:
The minor version does need to change, but I think the 9 reference should all be templated [here](https://github.com/stackhpc/stackhpc-kayobe-config/blob/stackhpc/yoga/etc/kayobe/pulp.yml#LL219C5-L219C5)

Reply:
I've updated the minor version and added a reno. I'm a little concerned about updating the default minor version but 1) we don't have many Rocky 9 deployments yet and 2) I'm assuming it won't cause any major issues